### ISSUE TITLE
Error when executing script

### ISSUE THREAD

**Issue by Remco-Veens** [KelvinTegelaar/SecureAppModel#3]
Hi,

When running this script i receive this error: As you can see i run it NOT in ISE.

Create-SecureAppModel.ps1 : Cannot bind argument
to parameter 'ObjectId' because it is null.
At line:1 char:1
+ .\Create-SecureAppModel.ps1
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidData: (:) [Create-SecureAppModel.ps1], ParameterBindingValidationException
    + FullyQualifiedErrorId : ParameterArgumentValidationErrorNullNotAllowed,Create-SecureAppModel.ps1

Can you help me?

**Comment by Pistoletjes** [comment]
Same here. I'm currently tracing the script to find the cause.

**Comment by KelvinTegelaar** [comment]
This only happens if it cant find the adminagents group, which only exists if you are a microsoft partner

**Comment by Pistoletjes** [comment]
$adminAgentsGroup = Get-AzureADGroup -Filter "DisplayName eq 'AdminAgents'"
    Add-AzureADGroupMember -ObjectId $adminAgentsGroup.ObjectId -RefObjectId $spn.ObjectId

@KelvinTegelaar yes, found that as well. It could use some proper error handling at those lines.

**Comment by KelvinTegelaar** [comment]
No, because the script is meant for just Microsoft partners that need the SAM. If you're not a partner you can just create any app via the portal.

**Comment by Pistoletjes** [comment]
Okay clear, thanks.
